Mapzen-ios-sdk.podspec

```diff
@@ -23,6 +23,6 @@ Pod::Spec.new do |s|
     cs.dependency 'OnTheRoad', '~> 1.0.0-beta'
     cs.dependency 'Tangram-es', '~> 0.4'
     cs.source_files = "src/*.swift"
-    cs.resources = 'images/*.png'
+    cs.resources = [ 'images/*.png', 'tangram/*' ]
```

Comment:
Ends up looking like this when imported. 
<img width="241" alt="screen shot 2017-02-01 at 10 32 42 am" src="https://cloud.githubusercontent.com/assets/1163880/22513303/d4b42cc4-e869-11e6-8336-f7d2642b7ea9.png">